dev/com.ibm.ws.logstash.collector_fat/fat/src/com/ibm/ws/logstash/collector/tests/ContainerEnvVarTest.java

```diff
@@ -0,0 +1,114 @@
+/**
+ *
+ */
+package com.ibm.ws.logstash.collector.tests;
+
+import static org.junit.Assert.assertTrue;
+
+import org.junit.After;
+import org.junit.AfterClass;
+import org.junit.Assume;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import com.ibm.websphere.simplicity.log.Log;
+
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.topology.impl.LibertyServer;
+import componenttest.topology.impl.LibertyServerFactory;
+
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class ContainerEnvVarTest extends LogstashCollectorTest {
+
+    private static LibertyServer server = LibertyServerFactory.getLibertyServer("ContainerEnvServer");
```

Comment:
Same as above, can just add `@Server("ContainerEnvServer")`, and not use `LibertyServerFactory.getLibertyServer("ContainerEnvServer");`